share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_v1/spack.yaml

```diff
@@ -1,5 +1,13 @@
 spack:
   view: false
+  include:
+  # Architecture configs for CI
+  - $spack/share/spack/gitlab/cloud_pipelines/configs/linux/neoverse-v1
+  # Platform configs for CI
+  - $spack/share/spack/gitlab/cloud_pipelines/configs/linux
+  # Base configs for CI
+  - $spack/share/spack/gitlab/cloud_pipelines/configs
+
```

Comment:
Would it be possible for `configs/linux/neoverse-v1` to automatically include `configs/linux`, which would then include `configs`? If so, that would reduce copy/paste across the stacks.